### PR TITLE
Expand on the current logger docs

### PR DESCRIPTION
**Pull request by Zen-CODE**
The current logger documentation does not really explain clearly what the Kivy logger offers and why you should use it. Because really you should when writing a kivy app....:-)

### CODE REVIEW

File: kivy/logger.py [hunk 1]
```diff
@@ -17,17 +25,40 @@
...
+some solid benefits and useful features. These include:
+
+* simplied usage (single instance, simple configuration, works by default)
+* color-coded output
+* output to `stdout` by default
+* message categorization via colon separation
+* access to log history even if logging is disabled
+* built-in handling of various cross-platform considerations
+
+For these reasons, we advise that you use the Kivy Logger unless you have good
```
Comment (matham) [inline kivy/logger.py:58]: I'm hesitant to push for it that strongly. I would rather state it indirectly - "that kivy's logger was designed to be used with kivy apps and it makes logging from such app more convenient", and leave it at that.

Comment (Zen-CODE) [reply kivy/logger.py]: Revised accordingly. Thanks

Comment (Zen-CODE) [reply kivy/logger.py]: @matham Bump :-)

**Comment by Zen-CODE** [comment]
Thanks :-)

**Pull request merged**